nodeJS/apis/api_basics.md

```diff
@@ -45,23 +46,23 @@ Each part of an API URI specifies the resource. For example, `GET /posts` would
 
 The [Same Origin Policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) is an important security measure that basically says "Only requests from the same origin (the same IP address or URL) should be allowed to access this API". (Look at the link above for a couple of examples of what counts as the 'same origin'.) This is a big problem for us because we are specifically trying to set up our API so that we can access it from different origins, so to enable that we need to set up Cross-origin resource sharing, or CORS.
 
-Setting up CORS in Express is very easy, there's a middleware that does the work for us. The official docs can be found [here](https://expressjs.com/en/resources/middleware/cors.html#enabling-cors-pre-flight).
+[Setting up CORS in Express](https://expressjs.com/en/resources/middleware/cors.html#enabling-cors-pre-flight) is very easy, there’s a middleware that does the work for us. 
 
 For now, it is acceptable to just allow access from any origin. This makes development quite a bit easier but for any _real_ project, once you deploy to a production environment you will probably want to specifically block access from any origin _except_ your frontend website. The documentation above explains how to do this.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1. [This article](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design) is a good resource for understanding and organizing RESTful APIs. If you want to code along with the first article, please note this includes the body-parser middleware to parse JSON data on the request body, however since Express 4.16.0 this parsing functionality has been incorporated directly into the express package itself.
-1. Read and code along with [this tutorial](https://www.robinwieruch.de/node-express-server-rest-api/) on setting up a REST API in Express. This is one of the best Express tutorials we've come across, it also talks about modular code organization, writing middleware, and links to some great extra info at the end.
+1. Read and code along with this tutorial on [setting up a REST API in Express](https://www.robinwieruch.de/node-express-server-rest-api/). If you want to code along with the first article, please note this includes the body-parser middleware to parse JSON data on the request body, however since Express 4.16.0 this parsing functionality has been incorporated directly into the express package itself.
+2. Read and code along with the following [tutorial](https://www.robinwieruch.de/node-express-server-rest-api/) to set up a REST API in Express. Discover one of the top Express tutorials covering modular code organization, middleware creation, and additional valuable resources.
```

Comment:
Not sure what happened here - the first link's href got changed by mistake, and the second link needs to be changed as per the spreadsheet.

```suggestion
1. Read about [RESTful API design](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design). If you want to code along with the first article, please note this includes the body-parser middleware to parse JSON data on the request body, however since Express 4.16.0 this parsing functionality has been incorporated directly into the Express package itself.
1. Read and code along with this tutorial on [setting up a REST API in Express](https://www.robinwieruch.de/node-express-server-rest-api/). This is one of the best Express tutorials we've come across, it also talks about modular code organization, writing middleware, and links to some great extra info at the end.

```